src/main/groovy/edu/wpi/first/gradlerio/wpi/WPIExtension.groovy

```diff
@@ -143,6 +144,7 @@ class WPIExtension {
                 "wpilibYear"           : new Tuple("WPILib Year", wpilibYear, "wpilibYear"),
                 "googleTestVersion"    : new Tuple("Google Test", googleTestVersion, "googleTest"),
                 "imguiVersion"         : new Tuple("ImGUI", imguiVersion, "imgui"),
+                "wpimathVersion"       : new Tuple("WPIMath", imguiVersion, "wpimath"),
```

Comment:
This doesn't seem right, the version variable for wpimath is the same as the imgui one.

Reply:
@ThadHouse I issued a PR to your fork, fixing this